defaults/main.yml

```diff
@@ -27,9 +27,10 @@
 # mdadm_arrays:
 # - name: 'md0'
 #   devices:
-#     - '/dev/sdb'
-#     - '/dev/sdc'
-#     - '/dev/sdd'
+#     - name: '/dev/sdb'
+#     - name: '/dev/sdc'
+#     - name: '/dev/sdd'
+#       additional_options: '--write-mostly'
```

Comment:
> -W, --write-mostly
```
subsequent devices listed in a --build, --create, or --add command will be flagged as 'write-mostly'. 
This is valid for RAID1 only and means that the 'md' driver will avoid reading
 from these devices if at all possible. This can be useful if mirroring over a slow link.
```

> --write-behind=
```
Specify that write-behind mode should be enabled (valid for RAID1 only). 
If an argument is specified, it will set the maximum number of outstanding writes allowed. 
The default value is 256. A write-intent bitmap is required in order to use write-behind mode, 
and write-behind is only attempted on drives marked as write-mostly.
```